views/layouts/main.handlebars

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
```

Comment:
it strikes me as a little funky to render this single page with a layout and specify `layout: false` on all the other pages? Maybe I'm missing how it's used more than once?

Reply:
yeah, I was just being a little lazy. Index / give use this layout, all the other views have their own. I think we can create a static page layout at some point too